src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Ethereum Energy Consumption
+description: The basic information you need to understand Ethereum's energy consumption.
+lang: en
+sidebar: true
+---
+
+Ethereum's current energy expenditure is too high and unsustainable. Resolving energy expenditure concerns without sacrificing security and decentralization has been a significant technical challenge for Ethereum's development. Let's explore why building Ethereum has had a high environmental impact and how upcoming network upgrades will dramatically change this.
```

Comment:
> Ethereum's current energy expenditure is too high and unsustainable.

Agree with the sentiment here but I'm not sure leading this page with a subjective statement is the right approach. It's "too high" - according to who? It's "unsustainable" - is it? Smells like someones opinion.

I'd suggest focusing more on factual statements. e.g. "Ethereum's current energy expenditure is significant - comparable to the power consumption of a small country". We could potentially link out to sources here (e.g. https://digiconomist.net/ethereum-energy-consumption/ though I'm not sure this is an authoritative source).

"Overall, Ethereum represents around 0.1% of global electricity" might be another useful stat for context:
https://kylemcdonald.github.io/ethereum-emissions/